Nur in gnome-vfs2-2.8.1.old/debian/patches: 12_user_visible_drives.patch.
Index: gnome-vfs-2.24.2/libgnomevfs/gnome-vfs-volume-monitor-daemon.c
===================================================================
--- gnome-vfs-2.24.2.orig/libgnomevfs/gnome-vfs-volume-monitor-daemon.c	2010-02-06 11:55:44.829288390 +0100
+++ gnome-vfs-2.24.2/libgnomevfs/gnome-vfs-volume-monitor-daemon.c	2010-02-06 11:55:48.097278145 +0100
@@ -988,6 +988,10 @@
 		vol->priv->is_user_visible = 1;
 		break;
 	default:
+		/* additionally mark devices below /media/ as user visible, in
+		 * case they are not recognized as devices above */
+		if (!strncmp (mount->mount_path, "/media/", 7))
+			vol->priv->is_user_visible = 1;
 		break;
 	}
 	
